### ISSUE TITLE
Colors from utox_theme.ini aren't parsed correctly

### ISSUE THREAD

**Issue by tsudoko** [notsecure/uTox#1005]
Contents of `utox_theme.ini`:
``` ini
MENU_BACKGROUND = 2d0e75
MENU_HOVER_BACKGROUND = 341186
LIST_BACKGROUND = 3a1397
LIST_HOVER_BACKGROUND = 4d19c7
```
Result:
![](http://a.pomf.se/mcytrj.png)
Expected result:
![](http://a.pomf.se/fxnocp.png)
uTox 7d06869, Gentoo x86_64
@stal888

**Comment by stal888** [comment]
oops

**Comment by GrayHatter** [comment]
fixed with #1006
